ros2topic/ros2topic/verb/find.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+from ros2topic.api import get_topic_names_and_types
+from ros2topic.verb import VerbExtension
+
+
+class FindVerb(VerbExtension):
+    """Output a list of available topics of a given type."""
+
+    def add_arguments(self, parser, cli_name):
+        parser.add_argument(
+            'topic_type',
+            help="Name of the ROS topic type to filter for (e.g. 'std_msg/msg/String')")
```

Comment:
Please add a completer for this argument. See other verbs for an example, e.g. https://github.com/ros2/ros2cli/blob/e5b0ceb73b5380a745a3dccba17c078ce42aa2c1/ros2topic/ros2topic/verb/info.py#L27-L28

Reply:
You beat me to it :)

Reply:
I guess `TopicNameCompleter` autocompletes topics' names, what about type ?

Reply:
Nevermind, found `TopicTypeCompleter` :+1:

Reply:
When looking for the `TopicNameCompleter` there is a similar completer for types: `TopicTypeCompleter`. It finds only the type for a given topic though (if a topic is given). Of no topic is given it uses `ros2msg.api.message_type_completer` which you can use directly for this use case.

Reply:
If no topic is passed to `TopicNameCompleter` it lists all topics types. That works too.

Reply:
Since you don't need the logic of `TopicNameCompleter` please call `ros2msg.api.message_type_completer` directly.

Reply:
Done :+1: